Dockerfile

```diff
@@ -2,6 +2,10 @@ ARG DISTRO=debian
 ARG VERSION=buster
 FROM $DISTRO:$VERSION
 
+# TODO - remove - Temporary workaround for Ubuntu archive being down
+RUN grep 'archive.ubuntu.com' /etc/apt/sources.list && sed -i.bak 's/archive.ubuntu.com/us.archive.ubuntu.com/' /etc/apt/sources.list; \
```

Comment:
-F here would be nice

Reply:
Yeh, mby. But it's literally only gonna be here for a few hours :)